src/resources/plugin.yml

```diff
@@ -5,7 +5,8 @@ description: ${project.description}
 
 main: ${mainClass}
 
-softdepend: [Multiverse-Core, My_Worlds, GriefPrevention, QuickShop, VanishNoPacket, Citizens, Votifier, Monolith, Minigames, MurderMystery, BuildBattle, VillageDefense, PrisonMines, ChatControlRed, RPlace]
+softdepend: [geSuitHomes, Multiverse-Core, My_Worlds, GriefPrevention, QuickShop, VanishNoPacket, Citizens, Votifier,
+             Monolith, Minigames, MurderMystery, BuildBattle, VillageDefense, PrisonMines, ChatControlRed, RPlace]
```

Comment:
Not sure if line wrapping is ok here.. might want to revert it to a single line?

Reply:
Tested this locally and it worked fine. Can revert to single line